tests/drop_column_with_index/run.sh

```diff
@@ -33,9 +36,30 @@ function run() {
 	check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
 
 	check_log_contain_with_retry "mock upstream instance restart" $WORK_DIR/worker1/log/dm-worker.log
-	# check_log_contain_with_retry "dispatch auto resume task" $WORK_DIR/worker1/log/dm-worker.log
 	check_log_contain_with_retry "meet error when read from local binlog, will switch to remote binlog" $WORK_DIR/worker1/log/dm-worker.log
 
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"query-status test" \
+		"get event returned an error" 1 \
+		"\"stage\": \"Paused\"" 1 \
+		"\"isCanceled\": false" 1
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"pause-task test" \
+		"\"result\": true" 1 \
+		"get event returned an error" 1
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"query-status test" \
+		"get event returned an error" 1 \
+		"\"stage\": \"Paused\"" 1 \
+		"\"isCanceled\": true" 1
+
+	sleep 5
```

Comment:
why sleep 5 here?

Reply:
auto resume interval 5 seconds

Reply:
we may reduce check interval and sleep twice as it

Reply:
If check-interval is too small, the task may auto resume before manually pause, and test will be unstable.